config/routes.rb

```diff
@@ -47,6 +47,9 @@
   get "/coronavirus-form/what-kind-of-transport" => "coronavirus_form/transport_type#show"
   post "/coronavirus-form/what-kind-of-transport" => "coronavirus_form/transport_type#submit"
 
+  get "/coronavirus-form/offer-advice" => "coronavirus_form/offer_advice#show"
+  post "/coronavirus-form/offer-advice" => "coronavirus_form/offer_advice#submit"
```

Comment:
I think these routes aren't necessary, since we have `expert_advice`

Reply:
ah great spot, i added those for the back button, i guessed the name wrong!
Removed.